build_deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE="quay.io/notifications-backend"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
+    echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN  must be set"
+    exit 1
+fi
+
+mvn clean package \
+    -DskipTests
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+
+docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
+docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" .
```

Comment:
If you use `docker` and not benefit from the Quarkus plugin, I guess it won't find the Dockerfile which lives in `src/main/docker/Dockerfile.jvm`

Reply:
`src/main/docker/Dockerfile.jvm` was used via Quarkus, when testing it with docker it looks for Dockerfile at the same path the build_script is located.

Reply:
I missed the file rename/move in the change log, all good